pkg/controller/cache/cache_controller.go

```diff
@@ -206,11 +198,12 @@ func (r *ReconcileCache) Reconcile(request reconcile.Request) (reconcile.Result,
 		instance.Status.ServiceName = serviceList.Items[0].Name
 		statusUpdate = true
 	}
-	statusUpdate = statusUpdate || instance.SetCondition("Ready", metav1.ConditionTrue, "")
+	conditionUpdate := instance.SetCondition("Ready", metav1.ConditionTrue, "")
```

Comment:
Oh, code optimizator prevents second expression to calculate it first time :)

Reply:
yep.
Well... I could probably just swap the `||` operands order